src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -0,0 +1,318 @@
+if (updateAction.isPresent()) {
+    sphereClient.execute(InventoryEntryUpdateCommand.of(inventoryEntry, updateAction.get()));
+}
+````
+
+#### `buildSetSupplyChannelAction`
+Used for comparision of `expectedDelivery` values from provided entries. `Optional` that may contain updating
+[SetSupplyChannel](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/commands/updateactions/SetSupplyChannel.java)
+action is returned.
+
+````java
+/*
+ * Having InventoryEntry instance with legacy data and InventoryEntryDraft with current data
+ * attempt to build "set supply channel" update action
+ */
+final Optional<UpdateAction<InventoryEntry>> updateAction =
+    InventoryUpdateActionUtils.buildSetSupplyChannelAction(inventoryEntry, inventoryEntrydraft);
+
+// If update action is present, the legacy entry can be updated by SphereClient instance.
+if (updateAction.isPresent()) {
+    sphereClient.execute(InventoryEntryUpdateCommand.of(inventoryEntry, updateAction.get()));
+}
+````
+
+#### `buildActions`
+Used for comparision of all field values from provided entries. `List` that may contain all necessary updating
+actions is returned. This method takes additional parameters of [InventorySyncOptions](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/inventories/InventorySyncOptions.java)
+and [TypeService](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/services/TypeService.java)
+because they are needed for custom fields comparision.
+
+
+````java
+/*
+ * Having InventoryEntry instance with legacy data and InventoryEntryDraft with current data
+ * attempt to build update actions. Also instances of InventorySyncOptions and TypeService
+ * have to be injected.
+ */
+final List<UpdateAction<InventoryEntry>> updateActions =
+    InventorySyncUtils.buildActions(inventoryEntry, inventoryEntrydraft, inventorySyncOptions, typeService);
+
+// After check if any update action is present, the legacy entry can be updated using SphereClient instance.
+if (!updateActions.isEmpty()) {
+    sphereClient.execute(InventoryEntryUpdateCommand.of(inventoryEntry, updateActions));
+}
+````
+
+## How does it work?
+
+The inventory sync uses the `sku` and `supplyChannel` key to match new inventory entries to existing ones.
+1. If an inventory entry exists with the same `sku` and a reference to a `supplyChannel` with the same key, it means that
+this inventory entry already exists on the CTP project. Therefore, the tool calculates update actions that should be
+done to update the old inventory entry with the new inventory entry's fields. Only if there are update actions needed,
+they will be issued to the CTP platform.
+2. If no matching inventory entry with the same `sku` and a reference to a `supplyChannel` with the same key is found,
```

Comment:
so if an inventory entry has the same sku but no reference or different reference, the tool will create a new one?